test/tests/functional/pbs_multi_sched.py

```diff
@@ -754,18 +742,30 @@ def test_pbfs_server_restart(self):
         n = self.scheds['sc1'].query_fairshare().get_node(name=str(TEST_USER))
         self.assertTrue(n.usage, 25)
 
-    def test_trim_fairshare(self):
+    def test_pbsfs_revert_to_defaults(self):
         """
-        Test pbsfs's trim fairshare tree functionality for multisched
+        Test if revert_to_defaults() works properly with multi scheds.
+        revert_to_defaults() removes entities from resource_group file and
+        removes their usage(with pbsfs -e)
         """
-        self.common_setup()
-        sched_name = 'sc1'
-        self.scheds[sched_name].add_to_resource_group(TEST_USER,
-                                                      11, 'root', 10)
-        self.scheds[sched_name].add_to_resource_group(TEST_USER1,
-                                                      12, 'root', 10)
-        self.scheds[sched_name].set_sched_config({'fair_share': 'True'})
-        self.scheds[sched_name].set_fairshare_usage(TEST_USER, 100)
+        self.setup_sc1()
+        a = {'queue_type': 'execution',
+             'started': 'True',
+             'enabled': 'True'}
+        self.server.manager(MGR_CMD_CREATE, QUEUE, a, id='wq1')
+
+        p1 = {'partition': 'P1'}
+        self.server.manager(MGR_CMD_SET, QUEUE, p1, id='wq1', expect=True)
```

Comment:
Combine the above two manager calls.  Put partition in the initial set of attributes.